lib/Hierarchy/styles.scss

```diff
@@ -0,0 +1,7 @@
+.hierarchy-line {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  width: 2px;
+  background: $neutral-light;
+}
```

Comment:
Added this in ready for the other lines work, I just didn't want to do another PR for this one class. Also suggestions for the name of the class is welcome.